buildlib/jucx/jucx-publish.yml

```diff
@@ -21,16 +25,24 @@ jobs:
         path: "we/need/to/go/deeper"
         # ^workaround agent issue with container in root path
 
+      - task: DownloadBuildArtifacts@0
+        displayName: Midterm artifacts get
+        inputs:
+          downloadType: 'specific'
+          itemPattern: 'libjucx_*.so/libjucx_*.so'
+          downloadPath: '$(System.DefaultWorkingDirectory)/bindings/java/resources'
+
       - bash: |
           set -eEx
           gcc --version
           ./autogen.sh
           ./contrib/configure-release --with-java
-        displayName: Configure
-
-      - bash: |
-          set -eEx
           make -s -j`nproc`
+          # Keep only libjucx_<platform>.so
```

Comment:
1. why need to delete them?
2. why need to build here? isn't it done during jucx-build.yml?

Reply:
The workflow was designed by @petro-rudenko and outlined here: https://github.com/openucx/ucx/pull/8796#issuecomment-1377733275
The purpose is to have a single JAR with both platforms supported.